examples/list-equality/list_equality.md

```diff
@@ -2,9 +2,9 @@
 
 We can use `==` and `!=` on lists to check deep equality of two lists: two lists are deep equal if they have the same members in the same order. Deep equality only works for `anydata` lists. `===` and `!==` check for the exact equality which matches the references of the lists.
 
-::: code updating_a_list.bal :::
+::: code list_equality.bal :::
 
-::: out updating_a_list.out :::
+::: out list_equality.out :::
 
 ## Related links
 - [Tuples - Ballerina by example](https://ballerina.io/learn/by-example/tuples)
```

Comment:
```suggestion
- [Tuples](https://ballerina.io/learn/by-example/tuples)
```

Reply:
IMO its better to keep the existing one to align with the `- Specification` and `- API documentation` suffixes in the existing BBEs

Reply:
@praneesha @hasithaa shouldnt we have any indication of what type of related link this is? for example, BBE, Spec orGuide ?

Reply:
Please refer https://ballerina.io/learn/by-example/websocket-service-payload-constraint-validation/